src/main/java/cruise/umple/sample/downloader/ConsoleMain.java

```diff
@@ -145,7 +146,7 @@ public static void main(String[] args) throws IOException {
                 data.getInputContent().ifPresent(content -> {
                   data.getOutputPath().getParent().toFile().mkdir();
 
-                  EcoreImportHandler handler = new EcoreImportHandler();
+                  UmpleImportHandler handler = new EcoreImportHandler();
```

Comment:
An interface UmpleImporterHandler, or part of the refactoring that we need to do in cruise.umple to create generic importers (i.e. some sort of factory to generate the correct Importer based on the provided data).